src/configure/configurers/ResourceSelectorFactory.ts

```diff
@@ -0,0 +1,18 @@
+import { TargetResourceType } from "../model/models";
+import { AksAzureResourceSelector } from "./AksAzureResourceSelector";
+import { IAzureResourceSelector } from "./IAzureResourceSelector";
+import { WebAppAzureResourceSelector } from "./WebAppAzureResourceSelector";
+
+export class ResourceSelectorFactory {
+
+    public static getAzureResourceSelector(targetType: TargetResourceType): IAzureResourceSelector {
+        switch (targetType) {
+            case TargetResourceType.WebApp:
+                return new WebAppAzureResourceSelector();
+            case TargetResourceType.AKS:
+                return new AksAzureResourceSelector();
+            default:
+                throw new Error("Not supported");
```

Comment:
define proper error message in Message.ts and use it here.

Reply:
updated